src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -280,22 +304,6 @@ const formConfig = {
               'ui:options': {
                 updateSchema: (formData, formSchema) => {
                   const fs = formSchema;
-                  if (
-                    get(
-                      'certifierRelationship.relationshipToVeteran.applicant',
-                      formData,
-                    )
-                  ) {
-                    // If the certifier is also an applicant, pre-populate first app slot with certifier info:
-                    populateFirstApplicant(
-                      formData,
-                      SIGNER_REL_PATH, // Used to verify we only ever apply this fn in one location
-                      formData.certifierName,
-                      formData.certifierEmail,
-                      formData.certifierPhone,
-                      formData.certifierAddress,
-                    );
-                  }
```

Comment:
Dropping this `updateSchema` functionality - the data replication has been moved into the `SignerContactInfoPage`'s custom `onGoForward` function.